src/modules/kvs-watch/kvs-watch.c

```diff
@@ -46,6 +46,7 @@ struct watcher {
     bool index_valid;           // flag if prev_start_index/prev_end_index set
     int prev_start_index;       // previous start index loaded
     int prev_end_index;         // previous end index loaded
+    int append_send_count;      // number of indexes loaded (for FLUX_KVS_STREAM)
```

Comment:
Probably this shouldn't be named "append" anything since there is no appending going on with FLUX_KVS_STREAM.  Should it be a `loaded_blob_count` instead?

Plural of index is indices.